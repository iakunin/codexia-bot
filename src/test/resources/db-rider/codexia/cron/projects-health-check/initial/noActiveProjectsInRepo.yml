codexia_project:
    -
        id: 1
        uuid: "groovy: UUID.randomUUID().toString()"
        external_id: 12
        coordinates: "test-project1/test-repo1"
        author: "some-first-author"
        author_id: 44
        deleted: "deleted1"
        project_created_at: "2019-12-20 12:01:02"
        created_at: "2019-12-21 15:00:00"
        updated_at: "2019-12-21 15:00:00"
    -
        id: 2
        uuid: "groovy: UUID.randomUUID().toString()"
        external_id: 34
        coordinates: "test-project2/test-repo2"
        author: "some-second-author"
        author_id: 55
        deleted: "deleted2"
        project_created_at: "2019-11-22 10:01:02"
        created_at: "2019-12-21 15:00:00"
        updated_at: "2019-12-21 15:00:00"
