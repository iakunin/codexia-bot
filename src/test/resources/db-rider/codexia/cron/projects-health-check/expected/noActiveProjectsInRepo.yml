codexia_project:
    -
        id: 1
        deleted: "deleted1"
    -
        id: 2
        deleted: "deleted2"
